src/components/Option.js

```diff
@@ -53,10 +49,17 @@ export default class Option extends Component {
     ]).isRequired,
   };
 
+  static contextTypes = {
+    isDisabled: PropTypes.bool.isRequired,
+    isHoveredValue: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.string,
+      PropTypes.number,
+    ]),
+  };
+
   static defaultProps = {
-    _isHovered: false,
     _isDisplayedAsSelected: false,
-    _isDisabled: false,
   };
 
```

Comment:
How about using just 'hovered' or 'disabled' like html and getting rid of prefix 'is'.